### PR TITLE
Dropdown free size was comupted incorrectly in case of vertical scrollbars

### PR DESCRIPTION
**Pull request by flongo**
Method `aria.utils.Size.setContrains` was not correctly taking into account the presence of a vertical scrollbar when the haight of the element is bigger than the maximum one allowed.

### CODE REVIEW

File: test/aria/widgets/form/autocomplete/popupWidth/AdaptToContentWidthTplScript.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    $classpath : "test.aria.widgets.form.autocomplete.popupWidth.AdaptToContentWidthTplScript",
+    $dependencies : ["test.aria.widgets.form.autocomplete.popupWidth.CustomResourceHandler"],
+    $destructor : function () {
+        this.handler.$dispose();
+    },
+    $prototype : {
+
+        getHandler : function () {
+            if (!this.handler) {
+                this.handler = new test.aria.widgets.form.autocomplete.popupWidth.CustomResourceHandler(this.$getId('messages'));
```
Comment (divdavem) [inline test/aria/widgets/form/autocomplete/popupWidth/AdaptToContentWidthTplScript.js]: Is the `this.$getId('messages')` parameter really useful? I don't understand where it is used.

File: test/aria/widgets/form/autocomplete/popupWidth/AdaptToContentWidthTest.js [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                        scope : testCase
+                    });
+                }
+            });
+        },
+
+        _afterDropdownOpen : function () {
+            this._checkDropdownWidth();
+            var field = this.getInputField("ac");
+            this.synEvent.execute([["type", field, "[back]a"]], {
```
Comment (divdavem) [inline test/aria/widgets/form/autocomplete/popupWidth/AdaptToContentWidthTest.js]: The `[back]` key is supported in the `aria.jsunit.RobotPhantomJS` implementation of the Robot but not in the implementations extending `aria.jsunit.RobotJava`. Maybe you should use `[BACK_SPACE]` instead.

**Comment by divdavem** [comment]
It seems to both work better and simplify the code. It is very good. Thank you for this PR.

**Comment by flongo** [comment]
Thank you for your comments @divdavem. I have applied the suggested changes, I will integrate as soon as the Travis builds terminates successfully

**Comment by divdavem** [comment]
It seems that changes introduced in commit 43add31528fe5e2baccf6471d7133356220bf1a4 make the `test.aria.widgets.container.dialog.resize.test1.DialogOnResizeTestCase` test fail on all browsers (apart from PhantomJS).

For example, with Chrome:
```
Assert #6 failed : offsetLeft should be around 50 instead of 41
Assert #7 failed : offsetWidth should be around 300 instead of 318
```

**Pull request merged**